### ISSUE TITLE
Please add correct version tags

### ISSUE THREAD

**Issue by akunszt** [balabit/syslog-ng-docker#29]
The balabit/syslog-ng:3.7 is actually contains the 3.14 version. Previously it contained 3.12 or 3.13 (don't remember exactly). It makes very hard to use it in production if the version tagged images are constantly changes.

```
$ docker run -it --rm balabit/syslog-ng:3.7 -V
syslog-ng 3 (3.14.1)
Config version: 3.14
Installer-Version: 3.14.1
Revision: 3.14.1-1
Compile-Date: Feb 27 2018 12:12:45
Module-Directory: /usr/lib/syslog-ng/3.14
Module-Path: /usr/lib/syslog-ng/3.14
Available-Modules: stardate,tfgetent,mod-python,system-source,redis,linux-kmsg-format,basicfuncs,pacctformat,kvformat,dbparser,appmodel,syslogformat,xml,confgen,sdjournal,graphite,afmongodb,afsmtp,csvparser,date,afsql,tags-parser,mod-java,geoip2-plugin,afstomp,affile,afamqp,afsocket,disk-buffer,riemann,json-plugin,cef,map-value-pairs,pseudofile,afprog,cryptofuncs,snmptrapd-parser,add-contextual-data,http,geoip-plugin,afuser
Enable-Debug: off
Enable-GProf: off
Enable-Memtrace: off
Enable-IPv6: on
Enable-Spoof-Source: on
Enable-TCP-Wrapper: on
Enable-Linux-Caps: on
Enable-Systemd: on
```

**Comment by mitzkia** [comment]
Hello, thanks for the feedback you are absolutely right. 
The balabit/syslog-ng:3.7 image contains an newer syslog-ng version.
We have deleted a 3.7 tag, and created a new tag: 3.14.1 for a new syslog-ng version.
So the latest syslog-ng will available in: balabit/syslog-ng:3.14.1

```
$ docker run -it balabit/syslog-ng:3.14.1 --version
Digest: sha256:e3b0b27fbbbc4e09a1338b226ba545ffcf86f1346fece2e62a5d0e90f18e0eb6
Status: Downloaded newer image for balabit/syslog-ng:3.14.1
syslog-ng 3 (3.14.1)
Config version: 3.14
Installer-Version: 3.14.1
Revision: 3.14.1-3
Compile-Date: Mar  2 2018 12:56:34
Module-Directory: /usr/lib/syslog-ng/3.14
Module-Path: /usr/lib/syslog-ng/3.14
Available-Modules: sdjournal,afstomp,afuser,geoip-plugin,cef,cryptofuncs,pacctformat,affile,xml,afsmtp,appmodel,afamqp,linux-kmsg-format,date,redis,stardate,add-contextual-data,map-value-pairs,json-plugin,basicfuncs,afsocket,tags-parser,riemann,graphite,csvparser,pseudofile,geoip2-plugin,afmongodb,confgen,system-source,mod-java,afsql,dbparser,syslogformat,tfgetent,disk-buffer,http,kvformat,mod-python,afprog,snmptrapd-parser
Enable-Debug: off
Enable-GProf: off
Enable-Memtrace: off
Enable-IPv6: on
Enable-Spoof-Source: on
Enable-TCP-Wrapper: on
Enable-Linux-Caps: on
Enable-Systemd: on
```

**Comment by akunszt** [comment]
Thank you.
